### PR TITLE
saxwriter attributes and values can have utf-8 non-ascii chars

### PR DESCRIPTION
**Pull request by mhrivnak**
fixes #2015 
https://pulp.plan.io/issues/2015

### CODE REVIEW

File: server/test/unit/plugins/util/test_saxwriter.py [hunk 1]
```diff
@@ -60,3 +62,17 @@ def test_short_empty_elements_false(self):
...
+    def test_utf8_writes(self):
+        """
+        Test that utf-8 non-ascii characters are handled without complaint.
+        """
+        xml_generator = XMLWriter(StringIO())
+
+        tag = u'𝅘𝅥𝅮'
+        xml_generator.startDocument()
+        xml_generator.writeDoctype('<!DOCTYPE string here>')
+        xml_generator.startElement(tag, {u'𝆑': u'𝆒'})
```
Comment (bmbouter) [inline server/test/unit/plugins/util/test_saxwriter.py]: I enjoyed this.

**Comment by mhrivnak** [comment]
I'm waiting to merge until the PR builder can run. If that passes and I'm not available, feel free to merge this.

**Comment by seandst** [comment]
ok test

**Comment by mhrivnak** [comment]
I'm not sure why they all failed. Looking at the f23 and el7 runs, they both have these two flake8-related lines:

```
15:49:42 Running flake8
15:49:52 ./server/test/unit/server/webservices/views/test_repositories.py:17:55: E275 missing whitespace after keyword
```

The thing it's complaining about has been there since March, and my local flake8 run on f23 doesn't complain about it. Hm.

**Comment by seandst** [comment]
ok test

**Comment by seandst** [comment]
only rhel5 is failing, and that's due to known issues with the rhel5 test runner.

**Pull request merged**